ecs/firehose.conf

```diff
@@ -0,0 +1,17 @@
+[INPUT]
+    Name forward
+    unix_path /var/run/fluent.sock
+
+[FILTER]
+    Name record_modifier
+    Match *
+    Record ec2_instance_id true
```

Comment:
This will add a key the logs like this
```
"ec2_instance_id": "true",
```

You need to use the AWS Filter to add the instance ID.